README.md

```diff
@@ -10,50 +10,84 @@
-The xGraph command line interface accelerates the building and running
-of xGraph Systems.
 
----
-#### Getting Started
-You will find all the necessary installation files in the `.\bin\`
-directory. To install the xgraph CLI, follow the instructions for your
-operating system found below.
+The xGraph package comes with two main features: the xGraph core and the xGraph 
+Command Line Interface.
 
-##### For Windows:
-You can use the `xgraph.msi` file to install the xgraph CLI. Simply double
-click the file and the installer will run.
 
-Additionally, you will have to add the xgraph path variable to your
-systems environment variable. This can be done for a single session
-through the command line, or you can set the environment variable
-permanently through windows settings.
+### xGraph Core
+The xGraph core architecture handles compiling, instantiating and communication 
+between xGraph systems. The xGraph core consists of Genesis and Nexus. Genesis 
+handles compilation of xGraph systems, while Nexus handles instantiation of and 
+communication between xGraph systems.
 
-The xgraph path variable is ``` {path to...}ProgramFiles/xGraph```.
-Append this to your your Environmental Path Variable ($PATH).
 
-To add the xgraph path variable to your system environment variable, go
-to "My Computer" > "Properties" > "Advanced" > "Environment Variables" > "Path"
-and add it to the list.
+### xGraph Command Line Interface (CLI)
+The xGraph command line interface accelerates the building and running of xGraph 
+Systems. Using the xGraph CLI, you can generate new modules and systems, compile 
+systems from a system structure object, and run systems from a system cache. 
 
-##### For Mac:
-You can use the `xGraph.dmg` file to install the xgraph CLI. Simply double
-click the file and the installer will walk you through the installation
-process.
 
-Because the package is unsigned, you may have to allow the installation
-of unsigned packages in the security control panel.
+#### Getting Started
+You can install xGraph using the [npm package manager](https://www.npmjs.com/). 
+Because xGraph is developed on the node.js platform, you will have to install 
+node.js, and the npm package manager which comes with it. This is easy to do 
+from the [node.js homepage](https://nodejs.org/en/). Once you have installed 
+node.js and npm, you can install xGraph using the package manager, or [download 
+the xGraph core from github](https://github.com/IntrospectiveSystems/xGraph) and 
+build the package locally.
+
+
+##### Install using npm package installer
+In your command line tool, use the following command to install xGraph core 
+globally using the npm package installer.
+```
+npm install -g xgraph
+```
 
-##### For Linux:
-Simply unpack the installation file (.tar or .gz) and restart your terminal
-session.
+If you are on a unix machine, you may have to use the `sudo` prefix.
+```
+sudo npm install -g xgraph
+```
 
+##### Install using npm package builder
```

Comment:
I would retitle to something more along the lines of "Building the Standalone with npm", as the actual build process does not use npm, and "npm package builder" isnt a thing. `npm run build` is just a macro for calling the build script with node. the actual library doing the compilation is [nexe](https://www.npmjs.com/package/nexe). as well, the resulting executables dont require node or npm to be installed afterwards

Reply:
https://docs.npmjs.com/cli/build npm build, builds a package... this isn't what we are using?

Reply:
in [package.json](https://github.com/IntrospectiveSystems/xGraph/blob/release/package.json#L7) we override that functionality.

as well, you dont show how to actually install anything here, only build it. installing the standalone requires you to either build an installer as well (which is currently not supported for the end user), or you need to put the resulting executable in a folder that is in your path. for example on linux, `/usr/bin`